game.pl

```diff
@@ -19,7 +13,11 @@
     pick_ball_to_move(BoardState, Player, Row, Column),
     pick_possible_move(BoardState, Player, Row, Column, Direction),
     move(BoardState, Player, Row, Column, Direction, NewBoardState),
-    not(is_game_over(Player, BoardState)),
+    not(is_game_over(Player, NewBoardState)),
+    display_board(BoardSize, NewBoardState),
```

Comment:
display new board before the game is over, like in computer turn

Reply:
Done